tests/unit/streamalert/alert_processor/outputs/test_aws.py

```diff
@@ -356,6 +356,26 @@ def test_add_single_attachment(self):
             # Raise an error if no payload of type MIMEApplication is found
             raise AssertionError
 
+    def test_different_attachment_name(self):
```

Comment:
i'd rename to `test_record_name_override` or something relevant to the record, as attachments are already tested by the previous piece